hathor/nanocontracts/runner/runner.py

```diff
@@ -296,13 +296,30 @@ def _unsafe_call_public_method(
             nc_args=nc_args,
         )
 
+        if method_name == NC_INITIALIZE_METHOD:
+            self._unsafe_post_initialize(
+                contract_id=contract_id,
+                blueprint_id=blueprint_id,
+                ctx=ctx,
+            )
```

Comment:
Why this call? Should we fail the contract creation in case any container has not been initialized?

Reply:
Got it. Thanks!